src/components/ListButtons.jsx

```diff
@@ -0,0 +1,45 @@
+import { useNavigate } from 'react-router-dom';
+const ListButtons = (props) => {
+	const navigate = useNavigate();
+
+	const buttonVariants = {
```

Comment:
Is there a reason for both variants having repeated classes? Or could those be part of the element's className?

Reply:
I found it visually cleaner to repeat here some bits of code, and then just assigning the element with ${buttonVariants[props.colorAdd]}.

But yes, there is no more reason than that, so this is something that can be changed.